source/Core/Threads/PIDThread.cpp

```diff
@@ -14,10 +14,11 @@
 #include "main.hpp"
 #include "power.hpp"
 #include "task.h"
-static TickType_t powerPulseRate        = 10000;
-static TickType_t powerPulseDuration    = 250;
-TaskHandle_t      pidTaskNotification   = NULL;
-uint32_t          currentTempTargetDegC = 0; // Current temperature target in C
+
+const TickType_t powerPulseWaitUnit     = 2500;

```

Comment:
Can you use the `TICKS_*` constants for the times please, sorry to make you do this change 😓

Reply:
Sorry, I wasn't aware of that constants! This seems to be equivalent to the previous code:
```
const TickType_t powerPulseWaitUnit  = 25*TICKS_100MS; // 2.5 s
static TickType_t powerPulseRate         = (5*TICKS_100MS)/2; // 250 ms
```

Reply:
Yep no problem at all; i put them in a while ago and missed a bunch of code when swapping over so slowly moving to them. Just trying to reduce the amount of magic numbers everywhere 😓